packages/console/src/components/Sidebar/index.tsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
+import Item from './components/Item';
+import Section from './components/Section';
+import { sections } from './consts';
+import Gear from './icons/Gear';
 import * as styles from './index.module.scss';
 
 const Sidebar = () => {
-  return <div className={styles.sidebar}>Sidebar</div>;
+  const { t: tSection } = useTranslation(undefined, {
+    keyPrefix: 'admin_console.tab_sections',
+  });
+  const { t: tItem } = useTranslation(undefined, {
+    keyPrefix: 'admin_console.tabs',
+  });
+
+  return (
+    <div className={styles.sidebar}>
+      {sections.map(({ title, items }) => (
+        <Section key={title} title={tSection(title)}>
+          {items.map(({ title, Icon }) => (
+            <Item
+              key={title}
+              title={tItem(title)}
+              icon={<Icon />}
+              isActive={title === 'api_resources'}
```

Comment:
How about setting the first item to active by default? Using item key could ended up not matching anything if we change the key in the future and forget to change this line together.

Reply:
it'll be replaced with route soon

Reply:
ok